README.md

```diff
@@ -60,6 +60,9 @@ Emoji | Represents | Links to | Comment
 📢 | Talks | the slides/recording/repo/etc.
 🎨 | Design | the logo/iconography/visual design/etc.
 👀 | Reviewed Pull Requests
+💵 | Financial Support | relevant page
+🔍 | Funding Finders
```

Comment:
I think "Grant Writers" might be a better word for this. But perhaps you can include both "Funding Fingers" and "Grant Finders" since some projects get funding from both corporate sponsors and non-profit foundations.